percona-mongo/1.10/README.md

```diff
@@ -0,0 +1,652 @@
+# Disaster Recovery
+
+<a name="troubleshooting"></a>
+# Troubleshooting
+
+TODO
+
+<a name="backup-options"></a>
+## Backup Options
+
+MongoDB Backups via DCOS are not supported in this Alpha release.
+
+We recommend [Percona-Lab/mongodb_consistent_backup](https://github.com/Percona-Lab/mongodb_consistent_backup) or [mongodump](https://docs.mongodb.com/manual/reference/program/mongodump/) is used to create manual backups against the DCOS service endpoint until backups are supported. Note: the '--oplog' flag should be added to the mongodump command for point-in-time consistency.
+
+<a name="restore-options"></a>
+## Restore Options
+
+MongoDB Restores via DCOS are not supported in this Alpha release.
+
+We recommend [mongorestore](https://docs.mongodb.com/manual/reference/program/mongorestore/) is used to restore backups manually against the service endpoint until restores are supported. Use the '--oplogReplay' flag with mongorestore to restore with point-in-time consistency.
+
+<a name="accessing-logs"></a>
+## Accessing Logs
+
+Logs for the scheduler and all service nodes can be viewed from the DC/OS web interface.
+
+- Scheduler logs are useful for determining why a node isn't being launched (this is under the purview of the Scheduler).
+- Node logs are useful for examining problems in the service itself.
+
+In all cases, logs are generally piped to files named `stdout` and/or `stderr`.
+
+To view logs for a given node, perform the following steps:
+1. Visit <dcos-url> to access the DC/OS web interface.
+2. Navigate to `Services` and click on the service to be examined (default _`PKGNAME`_).
+3. In the list of tasks for the service, click on the task to be examined (scheduler is named after the service, nodes are each `_NODEPOD_-#-node`).
+4. In the task details, click on the `Logs` tab to go into the log viewer. By default, you will see `stdout`, but `stderr` is also useful. Use the pull-down in the upper right to select the file to be examined.
+
+You can also access the logs via the Mesos UI:
+1. Visit <dcos-url>/mesos to view the Mesos UI.
+2. Click the `Frameworks` tab in the upper left to get a list of services running in the cluster.
+3. Navigate into the correct framework for your needs. The scheduler runs under `marathon` with a task name matching the service name (default _`PKGNAME`_). Service nodes run under a framework whose name matches the service name (default _`PKGNAME`_).
+4. You should now see two lists of tasks. `Active Tasks` are tasks currently running, and `Completed Tasks` are tasks that have exited. Click the `Sandbox` link for the task you wish to examine.
+5. The `Sandbox` view will list files named `stdout` and `stderr`. Click the file names to view the files in the browser, or click `Download` to download them to your system for local examination. Note that very old tasks will have their Sandbox automatically deleted to limit disk space usage.
+
+
+<a name="limitations"></a>
+# Limitations
```

Comment:
Please add all the limitations and missing features. Both from certification requirements and Percona requirements.